default/python/AI/InvasionAI.py

```diff
@@ -76,15 +78,25 @@ def get_invasion_fleets():
             planet_partial_vis_turn = universe.getVisibilityTurnsMap(pid, empire_id).get(fo.visibility.partial, -9999)
             if planet_partial_vis_turn < sys_partial_vis_turn:
                 continue
+            best_base_planet = INVALID_ID
+            best_trooper_count = 1  # won't bother using base troopers unless better than 1 per ship
```

Comment:
Well, OK, pretty much.  Checking for at least 1 trooper per ship was a very short-cut check to decide if the base troopers should be deemed a better choice than regular ships.  If you are only getting 1 per ship then I think it can easily be better to go with regular troopers.  I added TODO about doing more analysis, and moved the floor down to anything above zero, and made the later check a more complex, so it accepts 10 or fewer ships no matter how many troopers per ship, or up to 20 ships so long as it is more than 1 trooper per ship.